### ISSUE TITLE
ORM style functionality (or recommendation of a package that'd work well with it?) 

### ISSUE THREAD

**Issue by GeorgePearse** [JakobGM/patito#81]
I'd love to declare a patito model, and then with a bit more code be able to generate that table in a database.

I've considered both SQLModel and Ormar recently. 

Any recommendations for anything that might play nice?

**Comment by thomasaarholt** [comment]
That's interesting! Do you specifically mean to create an empty table in a database with the schema as described by patito?

**Comment by GeorgePearse** [comment]
Yeah, exactly that. Completely understand if you think it's something patito shouldn't have to worry about.

**Comment by GeorgePearse** [comment]
But in our repo I'll probably be adding 

.to_db() and from_db() methods to the patito models. 

and a create_db() would be nice. Especially given the fact that the other implementations I'd do would require very similar info.

**Comment by thomasaarholt** [comment]
Have a play and see what you come up with. If it's general enough, then I'm happy to have it added here!

**Comment by GeorgePearse** [comment]
Good plan, love the package in general, great work! Exactly what I was looking for.
